smartpy/optimizers/optimizer.py

```diff
@@ -0,0 +1,26 @@
+import theano.tensor as T
+
+
+class Optimizer(object):
+    def __init__(self, model, loss_fct, dataset, update_rules=None, param_modifiers=None):
```

Comment:
I don't think this is necessary in the `__init__` update_rules=None, param_modifiers=None